src/ploomber_engine/ipython.py

```diff
@@ -165,24 +165,31 @@ def __init__(self, nb, display_stdout=False):
         self._display_stdout = display_stdout
 
     @classmethod
-    def from_path(cls, path):
+    def from_path(cls, path, display_stdout=False):
         """Initialize client from a path to a notebook
 
-
         Parameters
         ----------
         path : str
             Path to the ``.ipynb`` file
 
+        display_output : bool, default=False
+            If True, it prints whatever output the notebook prints.
```

Comment:
If True, it prints whatever output the notebook prints. ->
If True, it prints the same output the notebook prints.

Reply:
fixed